### PR TITLE
Entity Work April 5 part 1

### PR DESCRIPTION
**Pull request by mmelndezlujn**
Required items, please complete
   
   Change(s):
   - Deleted Custom Entity mappings
   - Split hostname and domain
   - Split name and UPN suffix

   Reason for Change(s):
   - Needed to add full mappings

   Version Updated:
   - yes

   Testing Completed:
   - yes

   Checked that the validations are passing and have addressed any issues that are present:
   - no

### CODE REVIEW

File: Detections/CommonSecurityLog/Fortinet-NetworkBeaconPattern.yaml [hunk 1]
```diff
@@ -69,19 +69,19 @@ query: |
     | project StartTime, EndTime, SourceIP, DestinationIP, DestinationPort, BeaconCount, TimeDeltasInSeconds=list_list_TimeDeltainSeconds, Periodicity=series_stats_list_TimeDeltainSeconds_normalized_avg, ReceivedBytes=sum_ReceivedBytes, SentBytes=sum_SentBytes, Actions=set_set_DeviceAction
     // where periodicity is order of magnitude larger than time delta threshold (eliminates FPs whose periodicity is close to the values we ignored)
     | where Periodicity >= (10*TimeDeltaThresholdInSeconds)
-    | extend timestamp = StartTime, IPCustomEntity = DestinationIP
+    | extend timestamp = StartTime
```
Comment (shainw) [inline Detections/CommonSecurityLog/Fortinet-NetworkBeaconPattern.yaml:72]: Remove timestamp mapping also

File: Detections/DuoSecurity/TrustMonitorEvent.yaml [hunk 2]
```diff
@@ -14,23 +14,27 @@ query: |
...
-version: 1.0.2
+        columnName: surfaced_auth_access_device_ip_s
+version: 1.0.3
 kind: Scheduled
 metadata:
     source:
         kind: Community
     author:
-        name: SecurityJedi
+        name: Microsoft Security Research
```
Comment (shainw) [inline Detections/DuoSecurity/TrustMonitorEvent.yaml:37]: You can leave SecurityJedi, this is not from our internal research team.

File: Detections/PulseConnectSecure/PulseConnectSecureVPN-CVE_2021_22893_Exploit.yaml [hunk 3]
```diff
@@ -21,19 +21,19 @@ query: |
...
-version: 1.0.1
+        columnName: Source_IP
+version: 1.0.2
 kind: Scheduled
 metadata:
     source:
         kind: Community
     author:
-        name: Sittikorn S
+        name: Microsoft Security Research
```
Comment (shainw) [inline Detections/PulseConnectSecure/PulseConnectSecureVPN-CVE_2021_22893_Exploit.yaml:36]: You can leave Sittikorn S, this is not from our internal research team.

**Review by shainw — DISMISSED** [review]
Some inline comments, but for all please remove the timestamp mapping.  Anytime you see timestamp mapping, remove it.

**Comment by v-shukore** [comment]
Hello @mmelndezlujn Please resolve branch conflicts.

**Comment by v-shukore** [comment]
Hi @mmelndezlujn, could you please resolve branch conflicts.

**Comment by v-shukore** [comment]
Hi @mmelndezlujn, Please resolve branch conflicts. Thanks

**Review by ashwin-patil — APPROVED** [review]

**Pull request merged**